pages/1.10/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -89,6 +90,17 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [zk_master_credentials](#zk-master)          | [enterprise type="inline" size="small" /] The ZooKeeper master credentials.  |
 | [zk_agent_credentials](#zk-agent)           | [enterprise type="inline" size="small" /] The ZooKeeper agent credentials.  |
 
+[enterprise]
+### adminrouter_auth_cache_enabled
+[/enterprise]
+
+_This option was added in DC/OS 1.10.6._
+
+Controls whether the Admin Router authorization cache is enabled.
+
+*   `adminrouter_auth_cache_enabled: false` (default) Every authorization check Admin Router performs will load the user's permissions from the IAM.
+*   `adminrouter_auth_cache_enabled: true` Admin Router will cache the user's permissions for 5s after performing an authorization check.
```

Comment:
Can we have a space between `5` and `s` or write `5 seconds`? :) sorry for nitpicking, but a space between number and unit is standard :).